cobbler/actions/reposync.py

```diff
@@ -570,12 +575,18 @@ def yum_sync(self, repo):
             proxy = repo.proxy
         (cert, verify) = self.gen_urlgrab_ssl_opts(repo.yumopts)
 
-        # FIXME: These two variables were deleted
-        repodata_path = ""
-        repomd_path = ""
+        # Backport fix for
+        # https://github.com/cobbler/cobbler/issues/2865
```

Comment:
Personally it seems kind of odd to note the backport in a comment in the code.

Reply:
I will go ahead and remove the comment.  It doesn't really provide any real benefit.